service/src/main/kotlin/no/nav/su/se/bakover/service/sak/SakServiceImpl.kt

```diff
@@ -115,6 +127,41 @@ internal class SakServiceImpl(
         return sakRepo.hentSakForSøknad(søknadId)?.right() ?: FantIkkeSak.left()
     }
 
+    override fun opprettFritekstDokument(request: OpprettDokumentRequest): Either<KunneIkkeOppretteDokument, Dokument.UtenMetadata> {
```

Comment:
Jeg skjønner ikke helt intensjonen med denne servicen. Her returnerer vi et fire and forget brev? I hvilke tilfeller ønsker vi det?

Reply:
Vi valgte å bare kjøre på den basic, kjedelige måten å sende brev på. som vi gjør allerede i dag. et endepunkt for brevutkast, et endepunkt for å lagre og sende det ut